packages/clerk-js/src/ui/common/constants.ts

```diff
@@ -88,3 +88,5 @@ export const ERROR_CODES = {
   OAUTH_ACCESS_DENIED: 'oauth_access_denied',
   NOT_ALLOWED_ACCESS: 'not_allowed_access',
 };
+
+export const CLERK_IMAGE_BASE_URL = 'https://image.clerk.dev/';
```

Comment:
🔧 We need to cater for staging and local development enviroment.

Reply:
Right! PTAL.